.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+# Description
+
+Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+Additional snippets: [Thread](Paste the thread link here....)
+
+## Jira/Asana
+
+JIRA : `<Jira Link>`
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+## Checklist:
+
+- [ ] Add Jira ID(s) in PR title and in the description?
+- [ ] I have performed a self-review of my code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added positive and negative tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] Responsive design is handled if there are new components/UI changes?
+- [ ] Any Screenshots (mobile & desktop) required for PR? If yes, have you added the respective screenshots ?
+- [ ] Any manual dev testing done by you on beta/func environment?
+- [ ] Reviewer added (SLA 2 days)
+
+## Reviewer Checklist
+
+- [ ] PR has right logic changes and functionality with tests is implemented as expected - Confirmed by `<Reviewer>`
```

Comment:
This can be refined which is useful for every reviewer reviewing these PRs. For example, we can include 
- Sufficient QA/Dev-Testing is done with proof (test cases list)
- Ensure that the change in bundle size falls within the acceptable range.

You can omit the item that says it requires my approval... We will automate this and I'll enable my approval as mandatory in order to merge any PR.